lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -1809,7 +1809,7 @@ namespace Js
         //  3. If func is either undefined or null, return undefined.
         //  4. If IsCallable(func) is false, throw a TypeError exception.
         //  5. Return func.
-        BOOL result = JavascriptOperators::GetPropertyReference(handler, methodId, &varMethod, requestContext);
+        BOOL result = JavascriptOperators::GetPropertyReference(handler, methodId, &varMethod, GetScriptContext());
```

Comment:
I think the best thing to do is leave this as requestcontext and below have call to marshal it.

---
In reply to: [81822215](https://github.com/Microsoft/ChakraCore/pull/1648#discussion_r81822215) [](ancestors = 81822215)